### DISCUSSION TITLE
How to install latest ImageMagick on Ubuntu/Debian with apt?

### DISCUSSION THREAD

**Discussion by gerkas** [ImageMagick/ImageMagick#7003] [Help]
### ImageMagick version

7.1.1-25

### Operating system, version and so on

WSL Linux, Ubuntu 22.04 LTS, Debian

### Description

Hi all, 

Is there a way to install the latest IM7 using `sudo apt install imagemagick`? 

I've tried but using `apt` installs a 6.x version instead of 7.x. I've already sent `sudo apt update && apt upgrade`

Regards

**Comment by rustyscottweber** [comment]
I was looking at this as well. It doesn't seem like there is a regularly run build pipeline for .deb packages. The last .deb file was build in march 2023. However, even back then imagemagic version 7 was out. It looks like there is some hesitancy to update to the latest version because of the many packages that might depend on version 6.  Not ideal for you or I, but I get it. I haven't been able to fine a apt repository that I can add that contains the later version of imagemagick. Maybe someone else knows of one? Is there maybe a way to build a .deb file from the repo?
